test/jdk/com/sun/management/HotSpotDiagnosticMXBean/DumpThreads.java

```diff
@@ -677,13 +685,23 @@ private ThreadFields findThread(long tid, List<String> lines) {
     }
 
     /**
-     * Returns true if lines of a plain text thread dump contain the given text.
+     * Returns true if lines of a plain text thread dump containing the given text.
```

Comment:
Nit: Should it be: "...if lines of a plain text thread dump **is** containing" ?

Reply:
The original "contain" was correct. Alternatively:
> Returns true if lines is a plain text thread dump containing the given text.